scripts/configure.py

```diff
@@ -98,11 +98,15 @@
             'sudo systemctl enable rsyslog.service',
             'sudo systemctl restart rsyslog.service',
 
-            'echo "reconfiguring journald to only log to RAM"',
-            r'echo -e "[Journal]\nStorage=volatile\nRuntimeMaxUse=64M\nForwardToConsole=no\nForwardToWall=no\n" | sudo tee /etc/systemd/journald.conf',
+            'echo "If this is first deploy, reconfiguring journald to only log to RAM"',
+            r'[ ! -d /var/log/journal ] && echo -e "[Journal]\nStorage=volatile\nRuntimeMaxUse=64M\nForwardToConsole=no\nForwardToWall=no\n" | sudo tee /etc/systemd/journald.conf',
             'sudo systemctl enable systemd-journald.service',
             'sudo systemctl restart systemd-journald.service',
 
+            # Create directories and cronjobs for log persistence option
+            'sudo mkdir -p /var/log/journal',
+            'sudo systemd-tmpfiles --create --prefix /var/log/journal',
```

Comment:
If a directory does not exist, set logs to volatile
This directory is then made immediately afterwards, meaning that the first portion only works on first deploy and will not wipe out the user's `persist_logs` settings